db/memtable_list.h

```diff
@@ -128,11 +129,11 @@ class MemTableListVersion {
       LogBuffer* log_buffer);
 
   // REQUIRE: m is an immutable memtable
-  void Add(MemTable* m, autovector<MemTable*>* to_delete);
+  void Add(MemTable* m, autovector<MemTable*>* to_delete, size_t usage);
```

Comment:
The `usage` in `Add` and `TrimHistory` is the size of memtable being added. We pass that to `TrimHistory` so that when we can better track the most up-to-date size of memtable list. In the latest commit `usage` is removed from `Add` since we can get the size from `m`